R/pkg/R/mllib.R

```diff
@@ -694,8 +694,11 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, layers, blockSize = 128, solver = "l-bfgs", maxIter = 100,
+                   tol = 1E-6, stepSize = 0.03, seed = 0x7FFFFFFF) {
+            if (length(layers) <= 1) stop("layers vector require length > 0.")
```

Comment:
Perhaps as a small style nit even though it is not flagged by lint-r:
you might want to style this with bracket
```
if (...) {
}
```
like in [here](https://github.com/WeichenXu123/spark/blob/99d0e0ce8e3c94897dd7fb7f5c57c5d1b1c065e0/R/pkg/R/mllib.R#L187)